chain/stmgr/forks.go

```diff
@@ -174,9 +174,16 @@ func (us UpgradeSchedule) GetNtwkVersion(e abi.ChainEpoch) (network.Version, err
 
 func (sm *StateManager) HandleStateForks(ctx context.Context, root cid.Cid, height abi.ChainEpoch, cb ExecMonitor, ts *types.TipSet) (cid.Cid, error) {
 	retCid := root
-	var err error
 	u := sm.stateMigrations[height]
 	if u != nil && u.upgrade != nil {
+		migCid, ok, err := u.resultCache.Result(ctx, root)
+		if err == nil && ok {
+			log.Warnw("CACHED migration", "height", height, "from", root, "to", migCid)
```

Comment:
This probably doesn't warrant a warning.

Reply:
Ya, it's only a warning because everything else is a warning eg 'STARTING migration', 'COMPLETED migraiton'.

Reply:
Warning the first time we migrate makes sense, but warning when we fetch a cached migration doesn't.